tests/nfs/nfs_verify_enable_rootsquash_window_v3_linux_v4.py

```diff
@@ -0,0 +1,129 @@
+from time import sleep
+
+from nfs_operations import _enable_v3_mount, cleanup_cluster, setup_nfs_cluster
+
+from cli.ceph.ceph import Ceph
+from cli.exceptions import ConfigError, OperationFailedError
+from cli.utilities.utils import create_files, get_file_owner
+from cli.utilities.windows_utils import setup_windows_clients
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """Verify access file from both v3 (windows) mount and linux v 4.1 mount
+    Args:
+        **kw: Key/value pairs of configuration information to be used in the test.
+    """
+    # Get config
+    config = kw.get("config")
+
+    # nfs cluster details
+    nfs_nodes = ceph_cluster.get_nodes("nfs")
+    no_servers = int(config.get("servers", "1"))
+    if no_servers > len(nfs_nodes):
+        raise ConfigError("The test requires more servers than available")
+    servers = nfs_nodes[:no_servers]
+    port = config.get("port", "2049")
+    version = config.get("nfs_version", "4.1")
+    fs_name = "cephfs"
+    nfs_name = "cephfs-nfs"
+    nfs_export = "/export"
+    nfs_mount = "/mnt/nfs"
+    window_nfs_mount = "Z:"
+    fs = "cephfs"
+    nfs_server_name = [nfs_node.hostname for nfs_node in servers]
+    ha = bool(config.get("ha", False))
+    vip = config.get("vip", None)
+
+    # Squashed export parameters
+    nfs_export_squash = "/export_0"
+    original_squash_value = '"squash": "none"'
+    new_squash_value = '"squash": "rootsquash"'
+
+    # Linux clients
+    linux_clients = ceph_cluster.get_nodes("client")
+    no_linux_clients = int(config.get("linux_clients", "1"))
+    linux_clients = linux_clients[:no_linux_clients]
+    if no_linux_clients > len(linux_clients):
+        raise ConfigError("The test requires more linux clients than available")
+
+    # Windows clients
+    for windows_client_obj in setup_windows_clients(config.get("windows_clients")):
+        ceph_cluster.node_list.append(windows_client_obj)
+    windows_clients = ceph_cluster.get_nodes("windows_client")
+
+    try:
+        # Setup nfs cluster with v4.1 and v3
+        setup_nfs_cluster(
+            linux_clients,
+            nfs_server_name,
+            port,
+            version,
+            nfs_name,
+            nfs_mount,
+            fs_name,
+            nfs_export,
+            fs,
+            ha,
+            vip,
+            ceph_cluster=ceph_cluster,
+        )
+
+        # Enable V3 support
```

Comment:
With latest ceph build having fix for v3 prerequisite enabled in codebase, this is not required

Reply:
Thanks @msainiRedhat 
Updated